setup.py

```diff
@@ -21,7 +21,7 @@
         "microcosm>=2.12.0",
         "microcosm-flask>=2.8.0",
         "microcosm-logging>=1.5.0",
-        "microcosm-postgres>=1.19.0",
+        "microcosm-postgres>=2.2.0",
```

Comment:
this will fail CI until I release `2.2.0` pending review and merge of the related PR: https://github.com/globality-corp/microcosm-postgres/pull/102